### PR TITLE
Porting Shimmer

### PR DESCRIPTION
**Pull request by niels9001**
This PR ports the `Shimmer` control from the `Labs` repository: https://github.com/CommunityToolkit/Labs-Windows/tree/main/components/Shimmer

### CODE REVIEW

File: components/Shimmer/src/CommunityToolkit.WinUI.Controls.Shimmer.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
+  <PropertyGroup>
+    <ToolkitComponentName>Shimmer</ToolkitComponentName>
+    <Description>This package contains Shimmer.</Description>
+    <Version>8.0.0-beta.1</Version>
```
Comment (Arlodotexe) [inline components/Shimmer/src/CommunityToolkit.WinUI.Controls.Shimmer.csproj:5]: We should remove this line now that we have automatic versioning.
```suggestion
```

File: components/Shimmer/src/AdditionalAssemblyInfo.cs
Comment (michael-hawker) [inline components/Shimmer/src/AdditionalAssemblyInfo.cs:1]: This file is auto-generated now and can be removed.

**Review by michael-hawker — COMMENTED** [review]
Were we ready to bring this over, I thought we decided to hold on this for now?

**Comment by michael-hawker** [comment]
@niels9001 should we close this and then work on some more samples and scenarios in the Labs repo once we get it updated with new infrastructure?

**Comment by michael-hawker** [comment]
Some related discussion in https://github.com/CommunityToolkit/Windows/issues/95#issuecomment-1731825390

**Pull request closed without merging**